en/volume-snapshot-backup-restore-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Architecture of Backup and Restore Based on EBS Volume Snapshots across Multiple Kubernetes
+summary: Learn the architecture of backup and restore based on EBS volume snapshots in TiDB cluster deployed across multiple Kubernetes.
+---
+
+# Architecture of BR Federation
+
+We support [deploying TiDB clusters across multiple Kubernetes](deploy-tidb-cluster-across-multiple-kubernetes.md). BR Federation is designed to enable back up and restore the TiDB cluster deployed across multiple Kubernetes using EBS snapshots. Since we need to take snapshots of EBS volumes, only the TiDB Operator can access EBS volume information.
+But the TiDB Operator can only access the Kubernetes where it's deployed, it means a TiDB Operator can only back up TiKV data within its own Kubernetes. As a result, to complete EBS snapshot backup and restore across multiple Kubernetes, a coordinator role is required, and it is what the BR Federation plays. The overall architecture is as follows:
+
+![BR Federation architecture](/media/br-federation-architecture.png)
+
+In this architecture, BR Federation serves as the control plane, while each Kubernetes where TiDB components are deployed serves as the data plane. The control plane (BR Federation) and the data plane interact with each other through the Kubernetes API Server.
+BR Federation orchestrates `Backup` and `Restore` CRs in the data plane to complete backup and restore across multiple Kubernetes.
+
+# Backup Process
+
+## Backup Process in Data Plane
+
+The backup process in the data plane is mainly divided into three phases:
+
+Phase One: Make requests to PD to pause region scheduling and GC. This action suspends TiDB cluster region scheduling and GC to prevent data inconsistencies between TiKV instances at the moment of taking snapshot, because each TiKV instance may take snapshot at different time.
+Since TiDB components are interconnected across multiple Kubernetes clusters, this operation affects the entire TiDB cluster and only needs to be executed in one Kubernetes.
```

Comment:
```suggestion
1. **Phase One:** Request PD to pause region scheduling and Garbage Collection (GC). As each TiKV instance might take snapshots at different times, pausing scheduling and GC can avoid data inconsistencies between TiKV instances during snapshot taking. Since the TiDB components are interconnected across multiple Kubernetes clusters, executing this operation in one Kubernetes affects the entire TiDB cluster.
```